Gruntfile.coffee

```diff
@@ -569,39 +569,24 @@ module.exports = (grunt) ->
 			all:
 				src: [
 						# Root files
-						".*rc"
-						".editorconfig"
-						".eslint*"
-						".git*"
-						".*.{json,yml}"
-						".npmignore"
-						"*.{json,md}"
-						"Gruntfile.coffee"
-						"Licen?e-*.txt"
-						"Rakefile"
-
-						# Folders
-						"dep/**"
-						"script/**"
-						"site/**"
-						"src/**"
-						"theme/**"
+						"\.*"
+						"**"
 
 						# Exemptions...
+						"!node_modules/**"
+						"!lib/**"
+						"!dist/**"
```

Comment:
@Garneauma I'll add this suggestion, but feel free to add others and commit to them to the branch if you want. This can always get Squash-Merged in the end
```suggestion
						"!dist/**"
						"!.DS_Store"
```